### PR TITLE
Bump GraphQL to v16

### LINKED ISSUE #1252: Bump GraphQL dependency to v16
### Describe the bug

When using v16 of the [`graphql` dependency](https://npmjs.com/package/graphql), an error very similar to #380 appears when the SvelteKit app is run.

### Severity

serious, but I can work around it

### Steps to Reproduce the Bug

1. Create a SvelteKit app
2. Use v16 of the `graphql` dependency + Houdini
3. Run the app with `pnpm dev`
4. Note the `_enterList.apply` error in the console, this time with a stacktrace

### Reproduction

_No response_

### PR DESCRIPTION
**Pull request by radiden**
Fixes #1252

### CODE REVIEW

**Comment by SeppahBaws** [comment]
Thanks for taking the time to look into this!

It looks like the houdini-2.0 branch was out of date with main before you created this PR, which is causing github to be very confused. Could you try rebasing your changes off the latest commit on the houdini-2.0 branch?

**Comment by AlecAivazis** [comment]
i updated the houdini branch earlier today. i think this branch contains the commits but it shouldn't cause any issues, unless im missing something?

File: packages/houdini/src/codegen/transforms/paginate.ts [hunk 1]
```diff
@@ -702,19 +702,22 @@ function objectNode([type, defaultValue]: [
...
-	if (defaultValue) {
-		node.fields.push({
-			kind: graphql.Kind.OBJECT_FIELD,
-			name: { kind: graphql.Kind.NAME, value: 'default' } as graphql.NameNode,
-			value: {
-				kind: typeof defaultValue === 'number' ? 'IntValue' : 'StringValue',
-				value: defaultValue.toString(),
-			},
-		} as graphql.ObjectFieldNode)
+			...(defaultValue
```
Comment (AlecAivazis) [inline packages/houdini/src/codegen/transforms/paginate.ts:705]: Can we go back to the `if(defaultValue)` structure? the ternary here is a bit hard to read imo

**Review by AlecAivazis — CHANGES_REQUESTED** [review]

**Review by AlecAivazis — APPROVED** [review]
Thanks for pushing on this!

**Comment by SeppahBaws** [comment]
Whoops, didn't mean for this to be closed

**Comment by endigma** [comment]
Superseded by #1387

**Pull request closed without merging**